nodes.py

```diff
@@ -479,7 +485,8 @@ def append_manual_list(key: str, value: dict) -> dict[str, dict]:
 
     @staticmethod
     def download_model_info(path: Path | str | None, model_hash: str) -> dict | None:
-        print(f"ComfyUI-Image-Saver: Downloading model info. for '{model_hash if path is None else f"{Path(path).stem}:{model_hash}"}'.")
+        model_label = model_hash if path is None else f"{Path(path).stem}:{model_hash}"
+        print(f"ComfyUI-Image-Saver: Downloading model info. for '{model_label}'.")
```

Comment:
```suggestion
        print(f"ComfyUI-Image-Saver: Downloading model info for '{model_label}'.")
```
I'll just fix it here before I forget again :)